indexing-service/src/main/java/org/apache/druid/indexing/common/actions/RetrieveUnusedSegmentsAction.java

```diff
@@ -42,16 +43,21 @@ public class RetrieveUnusedSegmentsAction implements TaskAction<List<DataSegment
   @JsonIgnore
   private final Integer limit;
 
+  @JsonIgnore
```

Comment:
Unrelated to this PR:
This is a weird style that seems to be used in some task actions. Not sure why we even need `@JsonIgnore` here when we are already exposing a getter annotated with `@JsonProperty`. Other task actions, like `SegmentAllocateAction` don't do this.